styles/main.css

```diff
@@ -2,10 +2,40 @@
 @import "tailwindcss/components";
 @import "tailwindcss/utilities";
 
-@import './_hljs.css';
-@import './_markdown.css';
+@import "./_hljs.css";
+@import "./_markdown.css";
 @import "./_fonts.css";
 
 body {
   @apply font-overpass bg-white;
 }
+
+.flavour-switch {
```

Comment:
As stated above, for components there's no global style needed and we rely on Tailwind

Reply:
It's just a copy of the official Belt language switch with a custom class name. I'll apply proper styles 🙂